README.md

```diff
@@ -66,7 +70,7 @@ In your project locate this file `{{your_project}}_web.ex`, and add:
   defp html_helpers do
     quote do
       ...
-      use Phoenix.LiveView.Components.MultiSelect   ## <--- add this line
```

Comment:
Why this change?  The implementation of `use` calls the import with only one `[multi_select: 1]` function.

Reply:
I didn't work for me with `use` 🤷